ronaldo-alves/1.I/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Sistemas de informação</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+   <h1>Sistemas de informação Home Page</h1>
+   <h2>Sistemas de informação</h2>
+   <p>O curso de Sistemas de Informação possui matriz curricular atual, com professores atuantes no mercado profissional e altamente conceituados e titulados.<br/>
+      O curso capacita e proporciona oportunidades de carreira para os diplomados e que atendem plenamente aos anseios do mercado, sempre exigente.</p>
+   <h3>Atuação profissional</h3>
+   <p>O Profissional de Tecnologia da Informação tem a capacidade de compreender os problemas do mundo real,<br/>
+      aplicação dos conhecimentos adquiridos na solução de problemas, capacidade de análise sistêmica dos<br/>
+      problemas, capacidade de síntese e de realização de projetos e, finalmente, capacidade de avaliação<br/>
+      e julgamento. As habilidades pessoais desse profissional são: pensamento sistêmico, capacidade de<br/>
+      resolução de problemas, pensamento crítico, disciplina pessoal, persistência, curiosidade, capacidade<br/>
+      de auto aprendizado e aberto a mudanças. Habilidades Interpessoais: trabalho colaborativo, capacidade<br/>
+      de comunicação, capacidade para resolução conjunta de problemas. Conhecimentos Técnicos: abstração,<br/>
+      representação e organização da informação, uso de ferramentas computacionais para aplicação dos conhecimentos.</p>
+    <h4>Corpo docente</h4>
```

Comment:
Is it a sub-title of "Atuação profissional"?
Ref.: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements